esmvaltool/cmorizers/data/formatters/datasets/nsidc_0116_sh.py

```diff
@@ -15,10 +15,8 @@
     https://nsidc.org/data/NSIDC-0116
 
     Login required for download, but requires citation only to use
-
-
 """
-from .nsidc_common import cmorize
+from esmvaltool.cmorizers.data.formatters.nsidc_common import cmorize
 
 
```

Comment:
This script does not work. I guess the call to the cmorizer function should be changed.

Reply:
Fixed